README.md

```diff
@@ -28,6 +28,43 @@ export RACKSPACE_APIKEY=...
 docker-compose build && docker-compose up -d
 ```
 
+Alternatively, you can manually run each docker container with:
+
+```bash
+# startup content service dependencies
+docker run -d --name elasticsearch elasticsearch:1.7
+docker run -d --name mongo mongo:2.6
+
+# build and deploy the content service
+cd {wherever you have the deconst/content-service}
+docker build --tag content-service:1.0.0 .
+docker run -d -p 9000:8080 \
+              -e NODE_ENV=development \
+              -e STORAGE=memory \
+              -e MONGODB_URL=mongodb://mongo:27017/content \
+              -e ELASTICSEARCH_HOST=http://elasticsearch:9200/ \
+              -e ADMIN_APIKEY=$RACKSPACE_APIKEY \
+              --link mongo:mongo \
+              --link elasticsearch:elasticsearch \
+              --name content \
+              content-service:1.0.0 script/inside/dev
```

Comment:
You don't actually need to build the Docker images from source unless you've made changes locally; you can use the automated builds from quay, instead, and skip cloning any repos or running any `docker build` commands.

```bash
docker run -d -p 9000:8080 \
  # ... \
  quay.io/deconst/content-service

docker run -d -p 80:8080 \
  # ... \
  quay.io/deconst/presenter
```